cmd/horcrux/cmd/cosigner.go

```diff
@@ -165,11 +163,17 @@ func StartCosignerCmd() *cobra.Command {
 					Peers:     cosigners,
 				})
 
+				duration := fmt.Sprintf("%d%s", config.CosignerConfig.Timeout, "ms")
```

Comment:
so instead of setting it as a `ms` value by default, let user put in string of their choosing and in the config validation logic parse it as a duration

Reply:
Consider it done! :100: